LibTrixi.jl/src/api_jl.jl

```diff
@@ -4,7 +4,7 @@ function trixi_initialize_simulation_jl(filename)
 
     # Initialize simulation state
     # Note: we need `invokelatest` here since the function is dynamically upon `include`
-    simstate = invokelatest(Main.init_simstate)
+    simstate = Base.invokelatest(Main.init_simstate)
```

Comment:
```suggestion
    # Note: `invokelatest` is not exported until Julia v1.9, thus we call it through `Base`
    simstate = Base.invokelatest(Main.init_simstate)
```
Maybe as self-documentation for us?